packages/grid_client/scripts/compare_locked_balance.ts

```diff
@@ -0,0 +1,117 @@
+import TFGridGqlClient from "@threefold/graphql_client";
+import { ContractState } from "@threefold/gridproxy_client";
+
+import { BalanceGetModel, GridClient } from "../src";
+import { getClient } from "./client_loader";
+
+interface Contract {
+  contractID: string;
+  state: string;
+}
+
+async function getUsersWithContracts(grid: GridClient) {
+  const users: any[] = [];
+
+  const graphql = new TFGridGqlClient(grid.config.graphqlURL);
+
+  const [nodeContracts, nameContracts, rentContracts] = await Promise.all([
+    graphql.nodeContracts(
+      { twinID: true, contractID: true, state: true },
+      {
+        where: {
+          state_in: [ContractState.Created, ContractState.GracePeriod],
+        },
+      },
+    ),
+    graphql.nameContracts(
+      { twinID: true, contractID: true, state: true },
+      {
+        where: {
+          state_in: [ContractState.Created, ContractState.GracePeriod],
+        },
+      },
+    ),
+    graphql.rentContracts(
+      { twinID: true, contractID: true, state: true },
+      {
+        where: {
+          state_in: [ContractState.Created, ContractState.GracePeriod],
+        },
+      },
+    ),
+  ]);
+
+  const contracts = [...nodeContracts, ...nameContracts, ...rentContracts];
+
+  const twinIDs = [...new Set(contracts.map(contract => contract.twinID))];
+
+  const twins = await graphql.twins({ accountID: true, twinID: true }, { where: { twinID_in: twinIDs } });
+  const twinMap = new Map(twins.map(twin => [twin.twinID, twin.accountID]));
+
+  for (const contract of contracts) {
+    const accountID = twinMap.get(contract.twinID);
+    if (!accountID) continue;
+
+    let user = users.find(user => user.twinID === contract.twinID);
+
+    if (!user) {
+      user = {
+        twinID: contract.twinID,
+        accountID: accountID,
+        contracts: [],
+      };
+      users.push(user);
+    }
+
+    user.contracts.push({
+      contractID: contract.contractID,
+      state: contract.state,
+    });
+  }
+
+  return users;
+}
+
+async function getUserBalance(grid: GridClient, address: BalanceGetModel) {
+  const balance = await grid.balance.get(address);
+  return balance;
+}
```

Comment:
what is this method introduce? it just calls another method only

Reply:
Removed.